solidity/test/BondedECDSAKeepTest.js

```diff
@@ -1175,6 +1189,8 @@ contract("BondedECDSAKeep", (accounts) => {
         "Caller is not the keep member"
       )
     })
+
+    it("does not fail if slashing failed", async () => {})
```

Comment:
Missing implementation?

Reply:
I had problems with linter, I do not know what happens here. I removed this line. Test for this scenario is implemented [here](https://github.com/keep-network/keep-ecdsa/blob/77b95f9e47b908dc30503e315aa04e80b2f0c8ce/solidity/test/BondedECDSAKeepTest.js#L1020-L1042).